.github/workflows/generate-summary.yaml

```diff
@@ -410,14 +410,15 @@ jobs:
         run: |
           head -c 65000 issue.md > trimmed_issue.md
           if [ $(stat -c%s issue.md) -gt 65000 ]; then
+            echo "\n\`\`\`\nIssue text has been trimmed. Please check the following Gist Link for the untrimmed issue.\n" >> trimmed_issue.md
```

Comment:
use `printf` instead of `echo` since `echo` doesn't actually render escaped characters properly.
```
ewlu@ewlu:/scratch/ewlu/ci/gcc-precommit-ci$ echo "\n\`\`\`something"
\n```something
ewlu@ewlu:/scratch/ewlu/ci/gcc-precommit-ci$ printf "\n\`\`\`something"

```somethingewlu@ewlu:/scratch/ewlu/ci/gcc-precommit-ci$
```